config/environments/production.rb

```diff
@@ -113,3 +113,7 @@
   config.hosts << ENV.fetch('DOMAIN')
   config.hosts << ENV.fetch('STAGING_DOMAIN') # Domain used in Azure pipelines for deploying to the second slot
 end
+
+Raven.configure do |config|
+  config.dsn = ENV.fetch('SENTRY_DSN')
```

Comment:
Non-blocking: looking at `manage-courses-frontend`, I don't think configuration is needed; Sentry seems to pick it up on its own when that's the env variable used.

Reply:
Ah cool, since it's redundant I'll remove it!